contracts/test/integration/GatewayDiamondToken.t.sol

```diff
@@ -233,7 +232,7 @@ contract GatewayDiamondTokenTest is Test, IntegrationTestBase {
         vm.prank(address(saDiamond));
         vm.expectCall(recipient, abi.encodeCall(IIpcHandler.handleIpcMessage, (msgs[0])), 1);
         gatewayDiamond.checkpointer().commitCheckpoint(batch);
-        assertEq(token.balanceOf(recipient), 8);
+        assertEq(token.balanceOf(recipient), 0);
```

Comment:
why change this to 0?

Reply:
Because I have disable the transfer for Call kind.